src/utilities/__tests__/coerceValue-test.js

```diff
@@ -283,4 +284,39 @@ describe('coerceValue', () => {
       ]);
     });
   });
+
+  describe('for nested GraphQLList', () => {
```

Comment:
@jjergus Thanks for PR 👍 
I think it better to have `describe` for `for GraphQLList` and just implement all test from this table: http://facebook.github.io/graphql/June2018/#sec-Type-System.List